x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformTask.java

```diff
@@ -568,4 +576,45 @@ TransformTaskState getTaskState() {
         return context.getTaskState();
     }
 
+    public static PersistentTask<?> getTransformTask(String transformId, ClusterState clusterState) {
+        Collection<PersistentTask<?>> transformTasks = findTransformTasks(t -> t.getId().equals(transformId), clusterState);
+        if (transformTasks.isEmpty()) {
+            return null;
+        }
+        // Task ids are unique
+        assert (transformTasks.size() == 1);
```

Comment:
I know we do this all over the place (in fact I am wanting to fix it soon), but `assert` with no message can be VERY confusing.